Containers/apache/Dockerfile

```diff
@@ -1,59 +1,44 @@
 # Caddy is a requirement
 FROM caddy:2.6.2-alpine as caddy
 
-FROM debian:bullseye-20221219-slim
+FROM httpd:2.4.54-alpine3.17
 
 RUN mkdir -p /mnt/data; \
     chown www-data:www-data /mnt/data;
 
 VOLUME /mnt/data
 
-RUN set -ex; \
-    \
-    apt-get update; \
-    apt-get install -y --no-install-recommends \
-        apache2 \
-        supervisor \
-        wget \
-        ca-certificates \
-        openssl \
-        netcat \
-        dpkg-dev \
-    ; \
-    rm -rf /var/lib/apt/lists/*
+RUN apk add --no-cache bash tzdata coreutils supervisor wget ca-certificates openssl netcat-openbsd
 
 COPY --from=caddy /usr/bin/caddy /usr/bin/
 RUN chmod +x /usr/bin/caddy
 
-RUN a2enmod rewrite \
-    headers \
-    proxy \
-    proxy_fcgi \
-    setenvif \
-    env \
-    mime \
-    dir \
-    authz_core \
-    alias
-
-COPY nextcloud.conf /etc/apache2/sites-available/
-
-RUN rm /etc/apache2/ports.conf; \
-    sed -s -i -e "s/Include ports.conf//" /etc/apache2/apache2.conf; \
-    sed -i "/^Listen /d" /etc/apache2/apache2.conf
+RUN sed -i \
+		-e 's/^#\(LoadModule .*mod_rewrite.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_headers.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_proxy.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_proxy_fcgi.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_setenvif.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_env.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_mime.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_dir.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_authz_core.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_alias.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_mpm_event.so\)/\1/' \
+		-e 's/\(LoadModule .*mod_mpm_worker.so\)/#\1/' \
+		-e 's/\(LoadModule .*mod_mpm_prefork.so\)/#\1/' \
```

Comment:
I think prefork is not supported. See https://github.com/nextcloud/spreed/pull/8509/files

Reply:
Thats why we disable mpm_worker and mpm_prefork and enable mpm_event

Reply:
Ah I see. Sorry, didnt read correctly 😅